baselines/ppo2/ppo2.py

```diff
@@ -258,10 +265,15 @@ def learn(*, network, env, total_timesteps, early_stopping = False, eval_env = N
             if MPI is None or MPI.COMM_WORLD.Get_rank() == 0:
                 logger.dumpkvs()
 
+            # For TOM, every EVAL_FREQ updates we evaluate the agent with TOMs and BCs
+            if additional_params["OTHER_AGENT_TYPE"]  == "tom" \
+                    and update % additional_params["EVAL_FREQ"] == 0:
+                run_info = eval_and_viz_tom(additional_params, env, model, run_info)
+
             # Update current logs
             if additional_params["RUN_TYPE"] in ["ppo", "joint_ppo"]:
                 from overcooked_ai_py.utils import save_dict_to_file
-                save_dict_to_file(run_info, additional_params["SAVE_DIR"] + "logs")
```

Comment:
Also keep the old way of saving logs in order to not break old ppo plotting code and such.

Reply:
I made this change because the directory "SAVE_DIR" doesn't specify which seed is currently being used, and therefore logs.txt is overwritten for each seed. So I added additional_params["CURRENT_SEED_DIR"] to ppo.py (see here: https://github.com/HumanCompatibleAI/human_ai_coord/blob/7310cc722784dd1e103991032f67e0de945c2652/human_aware_rl/ppo/ppo.py#L406). Then log.txt saves to a different directory for each seed. What do you think?

Reply:
Yeah, you're right that the current way that it is just overwritten doesn't make much sense. I think that maybe we should remove the log saving from ppo2.py entirely, or keep the change you made and call it `temp_logs.txt` instead: effectively, we are already saving the same logs (it is literally the same variable) once training is complete [here](https://github.com/HumanCompatibleAI/human_ai_coord/blob/master/human_aware_rl/ppo/ppo.py#L436). You can choose what you think is best.

Reply:
Yes you're right that this is overkill at the moment. I think I'd prefer keeping the logs.txt for now though, because my workflow uses this .txt file rather than the pickle file saved [here](https://github.com/HumanCompatibleAI/human_ai_coord/blob/e4652c944eb1c654354f0fde582e20a31715aae2/human_aware_rl/ppo/ppo.py#L436). But I'll rename it to temp_logs.txt. Then, when I come back to using the ppo_toms.py code again for experiments I'll change my workflow to use the pickle file instead of the txt file. Then we won't need the log saving at all in ppo2.py, and I can delete it completely.

Let me know if you'd prefer a different solution to this.